nil pointer from pipelinerun describe

# Version and Operating System

**tkn Version:** 0.6.0

**Operating System:** alpine linux

# Expected Behavior

`tkn pipelinerun describe <name>` should show not `panic`.

# Actual Behavior

This error message is thrown:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x10de379]

goroutine 1 [running]:
k8s.io/apimachinery/pkg/apis/meta/v1.(*Time).Before(...)
        /workspace/pkg/mod/k8s.io/apimachinery@v0.0.0-20191004074956-01f8b7d1121a/pkg/apis/meta/v1/time.go:75
github.com/tektoncd/cli/pkg/cmd/pipelinerun.taskrunList.Less(...)
        /workspace/src/github.com/tektoncd/cli/pkg/cmd/pipelinerun/describe.go:222
sort.insertionSort(0x15d5e80, 0xc000569360, 0x0, 0x4)
        /usr/local/go/src/sort/sort.go:27 +0xc4
sort.quickSort(0x15d5e80, 0xc000569360, 0x0, 0x4, 0x6)
        /usr/local/go/src/sort/sort.go:209 +0x201
sort.Sort(0x15d5e80, 0xc000569360)
        /usr/local/go/src/sort/sort.go:218 +0x79
github.com/tektoncd/cli/pkg/cmd/pipelinerun.printPipelineRunDescription(0xc00040bd40, 0x7ffd93d66577, 0xe, 0x15ebf80, 0xc000395600, 0x8, 0x7ffd93d66500)
        /workspace/src/github.com/tektoncd/cli/pkg/cmd/pipelinerun/describe.go:146 +0x91a
github.com/tektoncd/cli/pkg/cmd/pipelinerun.describeCommand.func1(0xc0003d1080, 0xc0003fca50, 0x1, 0x3, 0x0, 0x0)
        /workspace/src/github.com/tektoncd/cli/pkg/cmd/pipelinerun/describe.go:121 +0x18a
github.com/spf13/cobra.(*Command).execute(0xc0003d1080, 0xc0003f06c0, 0x3, 0x4, 0xc0003d1080, 0xc0003f06c0)
        /workspace/pkg/mod/github.com/chmouel/cobra@v0.0.0-20191021105835-a78788917390/command.go:832 +0x460
github.com/spf13/cobra.(*Command).ExecuteC(0xc000391340, 0xc000395600, 0xc000391340, 0xc00009a058)
        /workspace/pkg/mod/github.com/chmouel/cobra@v0.0.0-20191021105835-a78788917390/command.go:920 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
        /workspace/pkg/mod/github.com/chmouel/cobra@v0.0.0-20191021105835-a78788917390/command.go:870
main.main()
        /workspace/src/github.com/tektoncd/cli/cmd/tkn/main.go:31 +0x4b
```

# Steps to Reproduce the Problem

1. Start a pipeline with multiple tasks in it
2. Immediately run a describe on it

--- a/pkg/cmd/pipelinerun/describe.go
+++ b/pkg/cmd/pipelinerun/describe.go
@@ -217,8 +217,16 @@ func newTaskrunListFromMap(statusMap map[string]*v1alpha1.PipelineRunTaskRunStat
 	return trl
 }
 
-func (s taskrunList) Len() int      { return len(s) }
-func (s taskrunList) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
-func (s taskrunList) Less(i, j int) bool {
-	return s[j].Status.StartTime.Before(s[i].Status.StartTime)
+func (trs taskrunList) Len() int      { return len(trs) }
+func (trs taskrunList) Swap(i, j int) { trs[i], trs[j] = trs[j], trs[i] }
+func (trs taskrunList) Less(i, j int) bool {
+	if trs[j].Status.StartTime == nil {
+		return false
+	}
+
+	if trs[i].Status.StartTime == nil {
+		return true
+	}
+
+	return trs[j].Status.StartTime.Before(trs[i].Status.StartTime)
 }

--- a/pkg/cmd/pipelinerun/describe_test.go
+++ b/pkg/cmd/pipelinerun/describe_test.go
@@ -809,3 +809,89 @@ tr-1   t-1         8 minutes ago   3 minutes   Succeeded
 
 	test.AssertOutput(t, expected, actual)
 }
+
+func TestPipelineRunDescribe_without_tr_start_time(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	trs := []*v1alpha1.TaskRun{
+		tb.TaskRun("tr-1", "ns",
+			tb.TaskRunStatus(
+				tb.StatusCondition(apis.Condition{
+					Type:   apis.ConditionReady,
+					Status: corev1.ConditionUnknown,
+				}),
+			),
+		),
+		tb.TaskRun("tr-2", "ns",
+			tb.TaskRunStatus(
+				tb.StatusCondition(apis.Condition{
+					Type:   apis.ConditionReady,
+					Status: corev1.ConditionUnknown,
+				}),
+			),
+		),
+	}
+
+	cs, _ := test.SeedTestData(t, pipelinetest.Data{
+		PipelineRuns: []*v1alpha1.PipelineRun{
+			tb.PipelineRun("pipeline-run", "ns",
+				cb.PipelineRunCreationTimestamp(clock.Now()),
+				tb.PipelineRunLabel("tekton.dev/pipeline", "pipeline"),
+				tb.PipelineRunSpec("pipeline"),
+				tb.PipelineRunStatus(
+					tb.PipelineRunStatusCondition(apis.Condition{
+						Status: corev1.ConditionUnknown,
+						Reason: resources.ReasonRunning,
+					}),
+					tb.PipelineRunTaskRunsStatus("tr-1", &v1alpha1.PipelineRunTaskRunStatus{
+						PipelineTaskName: "t-1",
+						Status:           &trs[0].Status,
+					}),
+					tb.PipelineRunTaskRunsStatus("tr-2", &v1alpha1.PipelineRunTaskRunStatus{
+						PipelineTaskName: "t-2",
+						Status:           &trs[0].Status,
+					}),
+					tb.PipelineRunStartTime(clock.Now()),
+				),
+			),
+		},
+		Namespaces: []*corev1.Namespace{
+			{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "ns",
+				},
+			},
+		},
+		TaskRuns: trs,
+	})
+
+	p := &test.Params{Tekton: cs.Pipeline, Clock: clock, Kube: cs.Kube}
+
+	pipelinerun := Command(p)
+	clock.Advance(10 * time.Minute)
+	actual, err := test.ExecuteCommand(pipelinerun, "desc", "pipeline-run", "-n", "ns")
+	if err != nil {
+		t.Errorf("Unexpected error: %v", err)
+	}
+	expected := `Name:           pipeline-run
+Namespace:      ns
+Pipeline Ref:   pipeline
+
+Status
+STARTED          DURATION   STATUS
+10 minutes ago   ---        Running
+
+Resources
+No resources
+
+Params
+No params
+
+Taskruns
+NAME   TASK NAME   STARTED   DURATION   STATUS
+tr-1   t-1         ---       ---        Running
+tr-2   t-2         ---       ---        Running
+`
+
+	test.AssertOutput(t, expected, actual)
+}

